src/services/events/events-router.ts

```diff
@@ -2,6 +2,8 @@ import { Router } from "express";
 import { StatusCodes } from "http-status-codes";
 import { publicEventValidator } from "./events-schema";
 import { Database } from "../../database";
+import { checkInUser } from "./events-utils";
```

Comment:
is this ever invoked

Reply:
yeah, I call the checkin function in this file